contracts/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "@openzeppelin/midnight-apps-contracts",
+  "version": "0.0.1-alpha.1",
+  "description": "OpenZeppelin Midnight Apps contracts",
+  "keywords": [
+    "openzeppelin",
+    "midnight-apps",
+    "midnight",
+    "contracts",
+    "security"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/OpenZeppelin/midnight-apps.git"
+  },
+  "license": "MIT",
+  "author": "OpenZeppelin Community <maintainers@openzeppelin.org>",
+  "bugs": {
+    "url": "https://github.com/OpenZeppelin/midnight-apps/issues"
+  },
+  "homepage": "",
+  "type": "module",
+  "exports": {
+    "./math/witnesses/*": {
+      "types": "./src/math/witnesses/*.ts",
+      "import": "./src/math/witnesses/*.js",
+      "default": "./src/math/witnesses/*.js"
+    },
+    "./math/utils/*": {
+      "types": "./src/math/utils/*.ts",
+      "import": "./src/math/utils/*.js",
+      "default": "./src/math/utils/*.js"
+    }
```

Comment:
The idea is that we're not shipping these, right? Or, at least not from this repo? If so, I'd remove the exports